src/robber_hideouts.py

```diff
@@ -2,34 +2,36 @@
 from .utils import log
 from .util_game import close_modal
 import time
-from .village import open_city
+from .village import open_village, open_city, open_map
 
-def robber_hideout_thread(browser: client, interval: int) -> None:
+def robber_hideout_thread(browser: client, village: int, interval: int, units: dict) -> None:
     while True:
-        robber = check_robber(browser)
-        if robber:
+        robber = check_robber(browser, village)
+        if robber:            
             outgoing_troops = check_troops(browser)
             if outgoing_troops:
-                open_city(browser)
-                log("Troops is busy right now.")
+                log("troops are busy right now.")
             else:
-                send_troops(browser, robber)
-                open_city(browser)
-                log("Troops sent.")
+                send_troops(browser, village, robber, units)
         else:
-            open_city(browser)
-            log("There is no Robber Hideout right now, will check again later.")
-
+            log("there is no Robber Hideout right now, will check again later.")
+        
         time.sleep(interval)
-        log("Refreshing the page.")
-        browser.refresh()
-        time.sleep(25)
+        #log("Refreshing the page.")
```

Comment:
you can just remove it if we dont need it anymore :)

Reply:
All right, I'd commented it out in first place because for me it didn't make sense reload the page after sending the troops, but this was already done at the initial version of the robber_hideouts, so not really sure which was the purpose meant for that. Furthermore, it also was causing problems because it was in the wrong place disturbing actions from other threads.